00_Base/src/interfaces/api/MessageQuerystring.ts

```diff
@@ -19,5 +19,17 @@
  */
 export interface IMessageQuerystring {
     identifier: string;
-    partyId: string;
+    tenantId: string;
```

Comment:
@joll90 Have you considered using constants for the keys? I have found this helpful in the past. For example:

```
export enum SomeObjectProperties {
  id = 'id',
  ...
}
export interface SomeObject extends SomeObjectBase {
  [SomeObjectProperties.id]?: string; // nullable
}
```
this way when you have to refactor, you can just go to the enum and change in one place and everywhere where you have tenantId can just be easily connected and updated

Reply:
Thanks @elliot-sabitov for bringing this up. Most of the places this `tentantId` is used is method signatures, only two interfaces really make use of this field name.